examples/models/object_detection/retina_net/basic/pascal_voc/train.py

```diff
@@ -112,7 +117,9 @@ def load_pascal_voc(
 Lets load some data and verify that our data looks as we expect it to.
 """
 
-dataset = load_pascal_voc(split="train", bounding_box_format="xywh", batch_size=9)
+dataset, dataset_info = load_pascal_voc(
```

Comment:
I think we should recommend returning ds_info from loaders just in case a user needs it. Keep in mind our downstream users will be writing loaders based on this as well.  Things like callbacks that visualize results using the id => class mapping, or learning rate schedules using cardinality, etc will all use this information.  So while unused in this script - I think it is a good habit to get in to return it.

Maybe I should assigned it to `_` instead?